app/src/org/commcare/utils/PhoneNumberHelper.java

```diff
@@ -82,31 +83,28 @@ public boolean isValidPhoneNumber(String phone) {
      */
     public int getCountryCode(String phone) {
         try {
-            Phonenumber.PhoneNumber phoneNumber = phoneNumberUtil.parse(phone, null);
-            if (phoneNumberUtil.isValidNumber(phoneNumber)) {
-                return phoneNumber.getCountryCode();
-            }
+            Phonenumber.PhoneNumber phoneNumber =
+                    phoneNumberUtil.parse(phone, "ZZ");
+            return phoneNumber.getCountryCode();
         } catch (NumberParseException e) {
-            // Ignore
+            return -1;
         }
-        return -1;
     }
 
     /**
      * Extracts the national phone number from a given full phone number.
      */
     public long getNationalNumber(String phone) {
         try {
-            Phonenumber.PhoneNumber phoneNumber = phoneNumberUtil.parse(phone, null);
-            if (phoneNumberUtil.isValidNumber(phoneNumber)) {
-                return phoneNumber.getNationalNumber();
-            }
+            Phonenumber.PhoneNumber phoneNumber =
+                    phoneNumberUtil.parse(phone, "ZZ");
+            return phoneNumber.getNationalNumber();
```

Comment:
can we explain the two changes here and how do they solve the issue in technical description for PR.